src/autocomplete/typeahead.js

```diff
@@ -363,7 +363,7 @@ _.mixin(Typeahead.prototype, {
   },
 
   getVal: function getVal() {
-    return this.input.getQuery();
+    return this.input.getInputValue();
```

Comment:
I don't know why [typeahead.js](https://github.com/twitter/typeahead.js/blob/master/src/typeahead/typeahead.js#L343-L345) is using `getQuery` here :/